tests/acceptance/features/coreApiSharees/sharees.feature

```diff
@@ -161,45 +161,6 @@ Feature: search sharees
       | 2               | 200        | 200         |
 
 
-  Scenario Outline: federated sharee for files
-    Given using OCS API version "<ocs-api-version>"
-    When user "Alice" gets the sharees using the sharing API with parameters
-      | search   | test@localhost |
-      | itemType | file           |
-    Then the OCS status code should be "<ocs-status>"
-    And the HTTP status code should be "<http-status>"
-    And the "exact users" sharees returned should be empty
-    And the "users" sharees returned should be empty
-    And the "exact groups" sharees returned should be empty
-    And the "groups" sharees returned should be empty
-    And the "exact remotes" sharees returned should be
-      | test@localhost | 6 | test@localhost |
-    And the "remotes" sharees returned should be empty
-    Examples:
-      | ocs-api-version | ocs-status | http-status |
-      | 1               | 100        | 200         |
-      | 2               | 200        | 200         |
```

Comment:
@ScharfViktor Should we remove these scenarios related to federated share (federated share is not implemented in oCIS yet but will be in the future: https://github.com/owncloud/ocis/issues/1329)?
IMO, these can be removed because these scenarios might not describe the federation share the way oCIS will implement it.

Reply:
these tests will not work in ocis when we implement federated share, please merge this PR
I left this test because I thought they would be able to work in ocis

Reply:
I think we can remove these. As we have a lot of tests in core regarding the federation that hasn't been transferred to ocis yet. When the feature will be implemented, we can copy and paste the implementation from core if the federate implementation in ocis works according to that. If the implementation in ocis will not work according to the implementation in core these scenarios aren't relevant anyway